### PR TITLE
Add kustomize tool to go.mod

### PR DESCRIPTION
**Pull request by honza**
Just like we did in #688 with controller-gen

### CODE REVIEW

File: config/render/capm3.yaml [hunk 1]
```diff
@@ -837,6 +837,16 @@ spec:
         control-plane: controller-manager
     spec:
       containers:
+      - args:
```
Comment (dhellmann) [inline config/render/capm3.yaml:840]: What happened here? Is this a generated file that changed because of a change in kustomize?

**Review by dhellmann — COMMENTED** [review]
This looks OK to me, with one question inline.

We should get @furkatgofurov7 and @kashifest to take a look, too.

File: config/render/capm3.yaml (hunk 1 above)
Comment (honza) [reply config/render/capm3.yaml:840]: Yes.

Comment (dhellmann) [reply config/render/capm3.yaml:840]: OK, thanks

Comment (furkatgofurov7) [reply config/render/capm3.yaml:840]: Yes, that's right.

**Pull request merged**